src/backend/distributed/executor/multi_real_time_executor.c

```diff
@@ -225,8 +225,7 @@ MultiRealTimeExecute(Job *job)
 	 */
 	if (taskFailed)
 	{
-		ereport(ERROR, (errmsg("failed to execute job " UINT64_FORMAT, job->jobId),
-						errdetail("Failure due to failed task %u", failedTaskId)));
+		ereport(ERROR, (errmsg("failed to execute task %u", failedTaskId)));
```

Comment:
What do you think about the repetition of the `ereport(ERROR, (errmsg("failed to execute task %u", failedTaskId)));` line. Would it make sense to move it to a function like `ShowFailedTaskError` or is it ok to keep it duplicated since it is only 1 LOC and nothing related to the logic.